airflow-core/src/airflow/ui/index.html

```diff
@@ -3,6 +3,8 @@
   <head>
     <meta charset="UTF-8" />
     <base href="{{ backend_server_base_url }}" />
+    <meta http-equiv="Content-Security-Policy"
+          content="frame-src 'self' {{ backend_server_base_url }};">
```

Comment:
```suggestion
          content="frame-src 'self';">
```

Reply:
`{{ backend_server_base_url }}` is a relative URL, I don't think that will actually do something

Reply:
![Screenshot 2025-06-26 at 15 43 30](https://github.com/user-attachments/assets/c0e69de7-8b2e-4dba-9538-38fc7fe80b18)